src/trace_processor/perfetto_sql/stdlib/wattson/tasks/threads_w_processes.sql

```diff
@@ -38,3 +40,120 @@ LEFT JOIN android_process_metadata AS package
   USING (upid)
 WHERE
   dur > 0;
+
+-- Gets all IRQs and labels each slice with hard or soft IRQ
+CREATE PERFETTO TABLE _all_irq_slices AS
```

Comment:
Can we expose this as a new, public table in the linux package? Feels like it belongs there.